docs/example-scenario/data/big-data-with-iot.md

```diff
@@ -0,0 +1,90 @@
+* Integrating and analyzing all construction site equipment and data to minimize equipment downtime and reduce theft.
+* Remotely and automatically controlling construction equipment to mitigate the effects of a labor shortage, ultimately requiring fewer workers and enabling  lower skilled workers to succeed.
+* Minimizing the operating costs and labor requirements for the supporting infrastructure, while increasing productivity and safety
+* Easily scaling the infrastructure to support increases in telemetry data.
+* Complying with all relevant legal requirements by provisioning resources in-country without compromising system availability.  
+* Using open-source software to maximize the investment in workers' current skills.
+
+Using managed Azure services such as IoT Hub and HDInsight will allow the customer to rapidly build and deploy a comprehensive solution with a lower operating cost. If you have additional data analytics needs, you should review the list of available [fully managed data analytics services in Azure][product-category].
+            
+## Potential use cases
+
+Consider this solution for the following use cases:
+
+* Construction, mining, or equipment manufacturing scenarios
+* Large-scale collection of device data for storage and analysis
+* Ingestion and analysis of large datasets 
+
+## Architecture
+
+![Architecture for IoT and data analytics in the construction industry][architecture]
+
+The data flows through the solution as follows:
+
+1. Construction equipment collects sensor data and sends the construction results data at regular intervals to load balanced web services hosted on a cluster of Azure virtual machines.
+2. The custom web services ingest the construction results data and store it in an Apache Cassandra cluster also running on Azure virtual machines.
+3. Another dataset is gathered by IoT sensors on various construction equipment and sent to IoT Hub.
+4. Raw data collected is sent directly from IoT Hub to Azure blob storage and is immediately available for viewing and analysis.
+5. Data collected via IoT Hub is processed in near real time by an Azure Stream Analytics job and stored in an Azure SQL database.
+6. The Smart Construction Cloud web application is available to analysts and end users to view and analyze sensor data and imagery. 
+7. Batch jobs are initiated on demand by users of the web application. The batch job runs in Apache Spark on HDInsight and analyzes new data stored in the Cassandra cluster. 
+
+### Components
+
+* [IoT Hub](/azure/iot-hub) acts as a central message hub for secure bi-directional communication between the cloud platform and the construction equipment and other site elements. IoT Hub can rapidly collect data for each device for ingestion into the data analytics pipeline.
```

Comment:
May be worth pointing out that IoT Hub provides per-device identity and secure communication between devices and cloud. Device identity is a big differentiator between IoT Hub and Event Hubs (or Kafka)